test/dynamo/test_misc.py

```diff
@@ -3144,6 +3144,52 @@ def forward(self, x):
         self.assertEqual(exported.device.type, "cpu")
         self.assertEqual(exported.dtype, torch.bfloat16)
 
+    def test_autocast_cpu_graph_break_nested(self):
+        class MyModule(torch.nn.Module):
+            @staticmethod
+            def mm_breaks(x, y):
+                # torch._dynamo.graph_break()
```

Comment:
should enable it?

Reply:
No, we currently can't support it, we will try to in https://github.com/pytorch/pytorch/pull/94672, I will change the test here to not require fixing inner function graph break

Reply:
Yea, let's fix [this pattern](https://github.com/pytorch/pytorch/issues/93890#issuecomment-1413236905) firstly and have a unit test for that.

Reply:
Ok, added.